Source/MetaAudienceNetworkAdapterBannerAd.swift

```diff
@@ -85,23 +85,27 @@ extension MetaAudienceNetworkAdapterBannerAd: FBAdViewDelegate {
 // MARK: - Helpers
 extension MetaAudienceNetworkAdapterBannerAd {
     private func fixedBannerSize(for requestedSize: BannerSize?) -> (size: CGSize, partnerSize: FBAdSize)? {
```

Comment:
Maybe we can simplify this a bit if we replace this function by a one that just maps BannerSize to FBAdSize, e.g.
```
private func fbAdSize(from size: BannerSize) -> FBAdSize {
        switch size {
        case .standard:
            kFBAdSizeHeight50Banner
        case .medium:
            kFBAdSizeHeight250Rectangle
        case .leaderboard:
            kFBAdSizeHeight90Banner
        default:
            kFBAdSizeHeight50Banner
        }
    }
```

We can then use it above like
```
guard let loadedSize = BannerSize.largestStandardFixedSizeThatFits(in: request.bannerSize ?? .standard),
    let partnerSize = fbAdSize(from: loadedSize) else ....
```

Reply:
For the record please see the Slack discussion [here](https://chartboost.slack.com/archives/C031DUA8DNE/p1714770207188179)

Reply:
[Done](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-meta-audience-network/pull/53/commits/f785483e25a9570c0573c2ac0f17fd75ebaf5622).

I really liked the variable name "fittingSize" - much better than "loadedSize" or the alternatives I'd considered.